lib/python/temporal/core.py

```diff
@@ -1427,31 +1427,33 @@ def init_dbif(dbif):
         If the database interface exists but is connected, the connection will
```

Comment:
Here the proper wording should be probably

```
If the database interface exists but is not connected, the connection will
```

Reply:
Comment updated. Thanks for your valuable insight.